jedi/evaluate/compiled/__init__.py

```diff
@@ -196,9 +203,12 @@ def name(self):
         return FakeName(name, self)
 
     def _execute_function(self, params):
+        from jedi.evaluate import docstrings
```

Comment:
This is necessary due to cyclical importing of CompliedObject in analysis.py which is import by docstrings.py through iteratable.py